packages/backend/src/plugins/catalog.ts

```diff
@@ -33,6 +33,10 @@ class OpenshiftResourceProcessor implements CatalogProcessor {
 
     if (originLocation.type === 'rhacm-managed-cluster') {
       resource.metadata.annotations ||= {};
+      resource.metadata.annotations['prometheus.io/alerts'] = 'all';
+      resource.metadata.annotations[
+        'prometheus.io/labels'
+      ] = `managed_cluster_id=${resource.metadata.annotations['cluster.open-cluster-management.io/id']}`;
```

Comment:
use the annotation from `ocm-common` instead.

Reply:
Should be resolved now waiting for https://github.com/janus-idp/backstage-plugins/pull/143 to get merged